packages/private/cross-account-stack-ref-handler/package.json

```diff
@@ -16,15 +16,29 @@
     "directory": "packages/private/cross-account-stack-ref-handler"
   },
   "scripts": {
-    "build": "parcel build src/index.ts --out-dir lib --target node --bundle-node-modules && yarn bundle",
+    "build": "parcel build src/index.ts --target node --cache-dir .cache && yarn bundle",
     "test": "jest",
     "format": "prettier --write **/*.ts !**/*.d.ts",
     "lint": "eslint . --ext .js,.ts",
     "clean": "rm -rf lib && rm -rf .cache",
     "bundle": "cd lib && rm -f bundle.zip && zip bundle.zip *"
   },
+  "targets": {
```

Comment:
Is there a way to test this lambda?

Reply:
um not really, we have tests, but the issue was a bundle not the code